README.md

```diff
@@ -1,94 +1,78 @@
+- ### To launch a new webview make this call in your application's JavaScript:
+      PGMultiView.loadView(url, strPayload, success, error);
 
-      - onActivityResult(requestCode, resultCode, intent )
-        - 'requestCode', an integer which is unpacked by the parent view from the intent passed by the childview
-        - 'result code', an arbitrary integer set in the parent view
-        > When the childview has been backgrounded (aka user navigates away from the PGMultiviewActivity.java) the parent view retrieves
-        the result of that activity ,"message from parent", and requestCode from the intent
+            @param 'url' the location of your html file.  The MultiView plugin automatically adds the prefix of :///android_asset/www/ if your url does not start with js
+            @param  'strPayload', data stored in a JSON object to be passed between views.  
+            @param 'success', a sucessful plugin callback
+            @param 'error', the plugin was not called
 
-      - onQuit()
-        During onQuit() the plugin is notified that the child view is being backgrounded.
-        Data and an identifying key are packaged into an intent and sent to the parent view
+- ### Please note that you must make two separate JavaScript files which correspond respectively to the native portions of the plugin(PGMultiview.java and PGMultiviewActivity.java) if you want to utilize both views.
 
-- Persisting Data
-
-      - Currently data is stored to memory (not disc).  Under most operating conditions this should not be an issue.
-       However, in an extreme low memory state you do risk losing data between views if either PGMultiView.java or or PGMultiViewActivity crash before reaching onStop() in the activity lifecycle.
-      - This should be resolved in the near future, the next release of this plugin bindsEvents() and writes the saved view state to local storage so that it can be retrieved at OnPause() in the Cordova Activities' lifecycle.
-
-ios:
-===
-
-Open a new view:
-====
-
-    PGMultiView.loadView(srcUrl);
+- ### To *dismiss* a webview make this call in your application's JS:
+      PGMultiView.dismissView(data);
 
-And remove it:
-====
+            @param 'data' is currently a string - will be JSON object in next release.
 
-    PGMultiView.dismissView();
+- ### The MultiView Sequence:
+![MultiViewSequence](MultiViewSequence.png)
 
+## Issues
+***
+Currently data is stored to memory (not disc).  Under most operating conditions this should not be an issue.
+However, in an extreme low memory state you do risk losing data between views if either PGMultiView.java or or PGMultiViewActivity crash before reaching onStop() in the activity lifecycle.
+This should be resolved in the near future, the next release of this plugin bindsEvents() and writes the saved view state to local storage so that it can be retrieved at OnPause() in the Cordova Activities' lifecycle.
 
-![MultiViewSequence](MultiViewSequence.png)
+## Contact Us
+***
```

Comment:
I think we can leave this section out.  For open source repos I think it is implied already.  Maybe just a 'pull requests welcome!', but I wouldn't do much more than that.